cmd/skaffold/app/cmd/fix.go

```diff
@@ -84,10 +86,25 @@ func fix(out io.Writer, configFile string, toVersion string, overwrite bool) err
 		if err := ioutil.WriteFile(configFile, newCfg, 0644); err != nil {
 			return fmt.Errorf("writing config file: %w", err)
 		}
-		color.Default.Fprintf(out, "New config at version %s generated and written to %s\n", cfg.GetVersion(), opts.ConfigurationFile)
+		color.Default.Fprintf(out, "New config at version %s generated and written to %s\n", cfg.GetVersion(), configFile)
+		warnForProfilePatchUpgrade(cfg, out)
```

Comment:
I think we should check for profile patches regardless of whether it's overwrite or send to stdout?

Reply:
discussed offline with @briandealwis. We can add this message as a comment to the upgraded config.